mod-gobi/mod-gobi.postman_collection.json

```diff
@@ -134,6 +134,108 @@
 								"description": "Create a new Organization with name GOBI, which is a vendor"
 							},
 							"response": []
+						},
+						{
+							"name": "Get or create Contributor name type",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"id": "26b15fda-7df4-473f-9ce4-ccfc4924460c",
+										"exec": [
+											"pm.test(\"GET Contributor Name types response is ok\", function () {",
+											"    pm.response.to.be.ok;",
+											"    let jsonData = pm.response.json();",
+											"    if (jsonData.contributorNameTypes.length == 1) {",
+											"        useAlreadyExistingType(jsonData.contributorNameTypes[0]);",
+											"    } else {",
+											"        createNewType();",
+											"    }",
+											"});  ",
+											"",
+											"function useAlreadyExistingType(materialType) {",
+											"    pm.test(\"Contributor Name Type already exists\", function () {",
+											"        pm.expect(materialType.id).to.exist;",
```

Comment:
Rename materialType to Contributor Name type

Reply:
Renamed to `existingType`